Tutorial for Registry import in SDK (#246)
latchbio/latch: docs/source/tutorials/registry_import.md

```bash
latch preview <path_to_workflow_directory>
```

The command will open up a new page in the browser that displays a preview of the sample sheet component.

![A preview of the sample sheet component on the workflow GUI](../assets/registry/samplesheet.png)

When you click "Import from Registry", a new import modal will pop up. 

![A preview of the sample sheet component on the workflow GUI](../assets/registry/import.png)

Here, you can select the table of interest and samples to be used in the workflow.

![A preview of the sample sheet component on the workflow GUI](../assets/registry/sample-selection.png)

**Important Note**:

* The types of columns in a Registry table must match one-to-one with the Python types of the data class defined in SDK. 
* For example, the Registry table in the picture above has three columns: **Name**, **r1**, and **r2**, which have the types **Text**, **File**, and **File** in Registry, respectively. Similarly, the property `name`, `r1`, and `r2` in the data class `Sample` has the Python types `str`, `LatchFile`, and `LatchFile`.

![A preview of the sample sheet component on the workflow GUI](../assets/registry/match-columns.png)

> To learn more about how to create columns with specific types in Registry, visit our [Registry Wiki here](https://latch.wiki/create-a-table).

Reviewer: ```suggestion
[See the wiki section on Registry types](https://latch.wiki/create-a-table) for more information.
```
kinda bad, needs to be much more specific
but that would need its own page probably

it also needs to explain how types match with Registry API types

Author: yeah I agree - I will make a new page. can we merge this for now so I can start sending to customers?
